profiles/common/modules/features/multisite_business_indicators/multisite_business_indicators_standard/tests/multisite_business_indicators_standard.test

```diff
@@ -217,13 +217,13 @@ class BusinessIndicatorsStandardQAWebTestCase extends DrupalWebTestCase {
    * Helper function to check user access.
    *
    * @param array $type
-   *   Can be either 'allow' or 'deny'
+   *   Can be either 'allow' or 'deny'.
    * @param object $account
-   *   Account to check permission on
+   *   Account to check permission on.
    * @param array $permissions
-   *   Array of permissions
+   *   Array of permissions.
    */
-  private function checkUserAccess($type, $account, $permissions) {
+  private function checkUserAccess(array $type = array(), $account = NULL, array $permissions = array()) {
```

Comment:
Looking at the code it seems that all 3 of these arguments are required, so do not provide default values for any of them.